src/contracts/Resources/services/fixture-services.yaml

```
@@ -0,0 +1,26 @@
+services:
+    Ibexa\Tests\Core\Repository\LegacySchemaImporter:
+        alias: 'test.ibexa.schema_importer'
+
+    test.ibexa.schema_importer:
+        class: Ibexa\Tests\Core\Repository\LegacySchemaImporter
+        public: true
+        arguments:
+            - '@doctrine.dbal.default_connection'
```

```suggestion
            $connection: '@doctrine.dbal.default_connection'
```
Question: don't we have here `ibexa.persistence.connection` service available? If we decide to do integration on multi repo setup, it can lead to unexpected issues.

---

Technically yes, we should have this service available. However, for multirepository setup, these will in itself be insufficient, as actually we would need a registry of all repositories and their respective connections to set all databases correctly.

I will apply the change and we'll see how it goes, but this will need to be improved for those cases.

---

Fixed in d8ca324a.